.circleci/config.yml

```diff
@@ -12,11 +12,15 @@ jobs:
     environment:
       COMPOSE_FILE: .circleci/docker-compose.circle.yml
       COMPOSE_PROJECT_NAME: redash
-    docker:
-      - image: circleci/buildpack-deps:xenial
+    machine: true
```

Comment:
Why?

Reply:
For including tests dir by mount /app/test/ due to .dockerignore.
https://circleci.com/docs/2.0/executor-types/#machine-executor-overview